src/components/Home.js

```diff
@@ -1,14 +1,35 @@
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
+import React, { useEffect, useState } from 'react';
+import axiosInstance from '../apis/axiosClient';
+import AlbumsList from './AlbulList';
 
 function Home() {
   const { status, user } = useAuth();
 
+  // testing axiosInstance request
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    axiosInstance
+      .get('/showMe')
+      .then((response) => {
+        setData(response.data);
+        console.log('this is axios instance' + data);
+      })
+      .catch((error) => {
+        console.error('Error fetching data:', error);
+      });
+  }, []);
+
+  // end of testing
```

Comment:
This testing stuff can be removed right? Might already be removed once you merge `main` into your feature branch

Reply:
Yes, I removed it